src/CovidAppointmentTable.js

```diff
@@ -65,22 +87,51 @@ export default function CovidAppointmentTable() {
 	const [onlyShowAvailable, setOnlyShowAvailable] = useState(true);
 
 	useEffect(() => {
+		resetData()
+	}, []);
+
+	useEffect(()=> {
+		setLoading(true)
+		setData(data => data.filter(entry => entry.latitude && entry.longitude)
```

Comment:
Filtering out any location without a latitude & longitude seems heavy-handed to me.

Reply:
This is one that I was debating. What should we show for locations with no lat/long? I would include them in the list, but I don't know if that's more confusing to the user.

Reply:
Silently removing things is more confusing than returning things adequately labeled.
Putting them last sounds good to me — do we have some expectation that this would be a frequent problem? Last I had looked everything came with an address.

Reply:
I put it there to catch the cases where the entry is missing the `latitude` or `longitude` coordinate since you cannot compute the distance from the user when there is a missing coordinate.

Reply:
Instead of filtering it out, we should just not compute the distance when lat/long is missing. And then if there is no `distance` key, don't show it in the UI.
That also would help with the case where you haven't entered a ZIP - no locations have a `distance` key and so the UI element doesn't appear.